### PR TITLE
Use howler

### PR DESCRIPTION
**Pull request by abelsonlive**
Implement Howler for audio playing and fix play/pause bug + distortion (#19 + #22 )

### CODE REVIEW

File: src/Player.js [hunk 1]
```diff
@@ -1,45 +1,79 @@
...
   componentDidMount() {
-    this.audioPlayer = document.getElementById('audio-player');
+    if (this.isSuspended()) {
+      this.setState({
+        playing: false
+      })
+    }
+  }
+
+  isSuspended = () => {
```
Comment (houseofaragon) [inline src/Player.js]: I would create this as an instance variable in componentDidMount
```this.isSuspended = window.Howler.state === 'suspended'```

then reference it in handlePlay and wherever else like 
```this.isSuspended```

**Comment by jak2030** [comment]
Why doesn't 

```javascript
  componentDidMount() {
    if (this.isSuspended()) {
      this.player.playing = true;
      this.setState({
        playing: true
      })
    }
  }
```

work?

File: src/Player.js [hunk 2]
```diff
@@ -1,45 +1,79 @@
...
     } else {
-      this.audioPlayer.pause();
+      this.handlePlay();
     }
-    this.setState({ paused: !this.state.paused});
-
   }
 
   render() {
     return (
```
Comment (houseofaragon) [inline src/Player.js]: not a big deal, more just an fyi: you could deconstruct state and props before the return, so youre not referencing this.state and this.props multiple times.

so you could add
```
const {playing} = this.state;
const {playerText, src} = this.props;
```
then reference below like:
```playing={playing}```

**Comment by houseofaragon** [comment]
works as expected in chrome, safari, mobile chrome (latest version)

does not autoplay in
firefox (latest version)

does not play sound in
mobile safari 11.1 (but this is an old version)

File: src/Player.js (hunk 2 above)
Comment (abelsonlive) [reply src/Player.js]: yeah jeremy was showing me that. why is it preferable to explicitly declare them vs. referencing via `this` ?

**Comment by abelsonlive** [comment]
RE: @jak2030  
>  I'm ok with hitting play to start, maybe just generally? it's kind of cool to see a page initiate and suddenly animate with the sound

I guess I kind of agree. I have had fun pressing play/pause on the first one and watching the water respond. It's kinda like we build these little static worlds that come to life when you press start the music. Also, If we're also going to eventually come up against loading time for 3D stuff (potentially unity apps etc), then maybe we should design the experience around "pressing play"?

**Comment by abelsonlive** [comment]
ok i'll merge this and we can figure out load times later.

**Pull request merged**